src/ImageSharp/Compression/Zlib/Crc32.cs

```diff
@@ -198,6 +208,103 @@ private static unsafe uint CalculateSse(uint crc, ReadOnlySpan<byte> buffer)
                 }
             }
         }
+
+#if NET5_0_OR_GREATER
```

Comment:
We'll need to actually target a .NET 5+ runtime to take advantage of this. I want to add an explicit target for .NET 6 for v3.0.0